CHANGELOG.md

```diff
@@ -11,10 +11,10 @@
 
 **Bug Fixes**
 
+- Fix when removing `collectionName` from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) [Breaking Change] ([marcosuma](https://github.com/marcosuma))
```

Comment:
I think `breaking change` should be the first thing in this item. I think we should also bold this text to make it stand out. Also a minor wording tweak to make the structure of this item align with the others.
```suggestion
- **Breaking Change**: Fixes an issue where removing `collectionName` from user role didn't work [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))
```